src/app/src/components/DashboardActivityReportToast.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect, useState } from 'react';
+import { arrayOf, string } from 'prop-types';
+
+import Snackbar from '@material-ui/core/Snackbar';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+
+const gradient = 'linear-gradient(-90deg, rgb(224, 32, 32) 0%, rgb(163, 79, 165) 21%, rgb(0, 145, 255) 51%, rgb(91, 201, 238) 79%, rgb(109, 212, 0) 100%)';
+
+const styles = {
```

Comment:
At narrower widths the styling on the Snackbar makes it expand horizontally leaving the little rainbow bar looking out of place

<img width="840" alt="Screen Shot 2021-02-15 at 12 16 24 PM" src="https://user-images.githubusercontent.com/17363/107985724-db925300-6f87-11eb-8c89-e79cc246d9f2.png">

Reply:
Updated styling to fix:
<img width="802" alt="Screen Shot 2021-02-16 at 11 04 26 AM" src="https://user-images.githubusercontent.com/21046714/108092116-60de3c00-704a-11eb-830e-1db0615fdc26.png">